content/reference/user/organization/index.md

```diff
@@ -25,48 +32,63 @@ Click on the pen icon to edit the **Organization name**.
 
 ## Credit consumption
 
-{{< img src="organization-credit.png" alt="Organization credit informations" >}}
+{{< img src="credits-consumption.png" alt="Organization credit informations" >}}
 
 * **Blue** - Available credits.
 * **Orange** - Consumed credits.
 
-### Admin users
+## Admin users
 
 Shows all System Admins in your organization.
 
-{{< img src="organization-admin-user.png" alt="Organization users admin" >}}
+{{< img src="admin-users.png" alt="Organization users admin" >}}
 
 For each System Admin, you will find their GitHub username, first name, and last name.
 
-### Credits
+## Credits
 
 {{< alert warning >}}
 This section is only available to System Admins.
 {{< /alert >}}
 
 Credits consumption history.
 
-{{< img src="organization-credits-row.png" alt="Organization credits view in row" >}}
+{{< img src="credits.png" alt="Organization credits view in row" >}}
 
 By clicking on a row, you will see all the details of the credit consumption for each month.
 
-{{< img src="organization-credits-detail.png" alt="Organization credits view in detail" >}}
+{{< img src="credits-detail.png" alt="Organization credits view in detail" >}}
 
 * **Type** - Which type of event occurred, with a link redirecting to the run when credits were used to run a simulation.
 * **Date** - The day the event took effect.
 * **Credits** - Number of credits gained or used from the event.
 
-### Plans
+## Plans
 
 {{< alert warning >}}
 This section is only available to the System Admin.
 {{< /alert >}}
 
 Plans view history.
 
-{{< img src="organization-plan.png" alt="Organization plan" >}}
+{{< img src="plans.png" alt="Organization plan" >}}
 
 * **Status** - Current status of the payment plan: **Terminated**, **Active**, or **PaymentFailure**.
 * **From** - Start date of the plan.
 * **To** - End date of the plan, if there is one.
 * **Credits** - Number of credits awarded each month by the plan.
+
+## Invitations
+
+{{< alert warning >}}
+This section is only available to the System Admin.
+{{< /alert >}}
+
+{{< img src="invitations.png" alt="Invitations" >}}
+
+* **Email** - Recipient of the invitation.
+* **Sent** - Date the invitation was sent.
+* **Expire** - End date of the invitation.
```

Comment:
After this date, the invitation will no longer be valid, and you"ll need to create manually a new invitation